zerver/views/streams.py

```diff
@@ -74,6 +76,42 @@ def add_default_stream(request, user_profile, stream_name=REQ()):
 
 @require_realm_admin
 @has_request_variables
+def create_default_stream_group(request, user_profile, group_name=REQ(),
+                                stream_names=REQ(validator=check_list(check_string))):
+    # type: (HttpRequest, UserProfile, Text, List[Text]) -> HttpResponse
+    streams = []
+    for stream_name in stream_names:
+        (stream, recipient, sub) = access_stream_by_name(user_profile, stream_name)
+        streams.append(stream)
+    do_create_default_stream_group(user_profile.realm, group_name, streams)
+    return json_success()
+
+@require_realm_admin
+@has_request_variables
+def update_default_stream_group(request, user_profile, group_name=REQ(), op=REQ(),
+                                stream_names=REQ(validator=check_list(check_string))):
+    # type: (HttpRequest, UserProfile, Text, Text, Text) -> HttpResponse
+    streams = []
+    for stream_name in stream_names:
+        (stream, recipient, sub) = access_stream_by_name(user_profile, stream_name)
+        streams.append(stream)
```

Comment:
I'd add a blank line after this to separate the sections of this function (it'll be more readable)